cl/corpus_importer/management/commands/normalize_judges_opinions.py

```diff
@@ -0,0 +1,79 @@
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.people_db.lookup_utils import (
+    lookup_judge_by_last_name,
+    lookup_judges_by_last_name_list,
+)
+from cl.search.models import Opinion, OpinionCluster
+
+
+class Command(VerboseCommand):
+    help = (
+        "Populate author field in opinions using author_str field and "
+        "populate panel in opinion clusters using judges field. "
+    )
+
+    def handle(self, *args, **options):
+
+        self.normalize_authors_in_opinions()
+
+        self.normalize_panel_in_opinioncluster()
+
+    def normalize_authors_in_opinions(self):
```

Comment:
You should move this and your other method to be functions, like in the command to load from the API. If you do that, your tests get easier.